extensions/_modules/boto_ec2.py

```diff
@@ -1813,7 +1813,7 @@ def detach_volume(volume_id=None, instance_id=None, device=None, force=False,
         except boto.exception.BotoServerError as e:
             log.error(e)
             return False
-    elif tags:
+    elif filters:
         try:
             volumes = conn.get_all_volumes(filters = {'tag:{0}'.format(tag_name):'{0}'.format(tag_value)})
```

Comment:
They are part of salt-call:
`salt-call boto_ec2.detach_volume filters='{"tag:<tag_name>":"<tag_value>"}'`